### PR TITLE
metafeed/add/derived versus metafeed/add/existing

### LINKED ISSUE #22: Nonce field should not exist for adding existing "main" feed
In [Existing SSB Identity](https://github.com/ssb-ngi-pointer/ssb-meta-feed-spec#existing-ssb-identity) case, we have

```
{
  "type" => "metafeed/add",
  "feedpurpose" => "main",
  "subfeed" => (BFE-encoded feed ID for the 'main' feed),
  "metafeed" => (BFE-encoded Bendy Butt feed ID for the meta feed),
  "tangles" => {
    "metafeed" => {
      "root" => (BFE nil),
      "previous" => (BFE nil)
    }
  }
}
```

However, in [New SSB Identity](https://github.com/ssb-ngi-pointer/ssb-meta-feed-spec#new-ssb-identity) we have

```
{
  "type" => "metafeed/add",
  "feedpurpose" => "main",
  "subfeed" => (BFE-encoded feed ID for the 'main' feed),
  "metafeed" => (BFE-encoded Bendy Butt feed ID for the meta feed),
  "nonce" => (bencode byte sequence with 32 random bytes),
  "tangles" => {
    "metafeed" => {
      "root" => null,
      "previous" => null
    }
  }
}
```

Note that in the **second we have nonce** and in the **first case we don't**, but the message type is "metafeed/add" in both cases.

**This is a gotcha**, currently [JS ssb-meta-feeds is wrong](https://github.com/ssb-ngi-pointer/ssb-meta-feeds/blob/ec4824f32f4fab016e6ca81c2e02da83485630b3/messages.js#L57-L58) because it has a nonce for the main feed, which is wrong, because the nonce has nothing to do with deriving the main keys.

cc @arj03 @cryptix 

cc @mycognosist for validation

### PR DESCRIPTION
**Pull request by staltz**

### CODE REVIEW

**Review by arj03 — APPROVED** [review]
Looks good

**Review by cryptix — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -44,14 +44,15 @@ The `content` dictionary inside the `contentSection` of meta feed messages
...
+   - `metafeed/add/existing`
+   - `metafeed/add/derived`
    - `metafeed/update`
    - `metafeed/tombstone`
  - Has a `subfeed` field mapping to a BFE "feed ID", i.e. `<00> + format + data`
  - Has a `metafeed` field mapping to a BFE "Bendy Butt feed ID", i.e.
  `<00 03> + data`
- - (Only if the `type` is `metafeed/add`): a `nonce` field mapping to 32 random
- bytes in bencode
+ - (Only if the `type` is `metafeed/add/derived`): a `nonce` field mapping
```
Comment (cryptix) [inline README.md]: wondering if we should mention that it should be prefixed with `0x0603` per the new binary BFE thingy?

Comment (staltz) [reply README.md]: Yeah!

**Pull request merged**